vendors/shopgate_library/classes/apis.php

```diff
@@ -1280,6 +1418,29 @@ private function _getFileMeta($file, $parentLevel = 0) {
 
 		return $meta;
 	}
+
+	/**
+	 * enable error reporting to show exeption on request
+	 */
+	private function setEnableErrorReporting(){
+		@error_reporting(E_ERROR | E_CORE_ERROR | E_USER_ERROR);
+		@ini_set('display_errors', 1);
+	}
+
+	/**
+	 * @param $errorFile
+	 */
+	private function setEnablePrintErrorsToLog($errorFile){
+		$logFileHandler = @fopen($errorFile, 'a');
+		@fclose($logFileHandler);
+		@chmod($errorFile,0777);
+		@chmod($errorFile,0755);
+		@error_reporting(E_ALL ^ E_DEPRECATED);
+		@ini_set('log_errors', 1);
+		@ini_set('error_log', $errorFile);
+		@ini_set('ignore_repeated_errors', 1);
+		@ini_set('html_errors', 0);
+	}
```

Comment:
Please do not redirect the whole PHP error output to a specific file. If you really want to store this kind of data, make your own exception handler which will write defined messages to the file you want.